ConvertCSVToXmlOrchard/ConvertCSVToXmlOrchard/CsvToXml.cs

```diff
@@ -64,20 +64,22 @@ private static void SaveToDataBase(XElement content)
                 XElement ValueFromDatabase = XElement.Load(databasepath);
                 var loaddata = ValueFromDatabase.Descendants("Content");
                 loaddata.Remove();
-                if (loaddata.Any()) 
+                if (loaddata.Any())
                 {
 
                 }
-                else 
+                else
                 {
-                    ValueFromDatabase.Add(content); 
+                    ValueFromDatabase.Add(content);
                     ValueFromDatabase.Save(databasepath);
                 }
 
             }
             else
             {
-                var ValueFromDatabase = XElement.Load(listOfValueFromDatabase[0]); 
+                XDocument ValueFromDatabase = new XDocument();
+                ValueFromDatabase = new XDocument(new XDeclaration("1.0", "UTF-8", "yes"));
+                ValueFromDatabase.Add(new XComment("Exported from Orchard"),XElement.Parse(listOfValueFromDatabase[0]));
```

Comment:
That is not really true ;-)
This XML file is "Created by Fai", not "Exported from Orchard" :grimacing: